src/pages/ReferralDetailsPage.js

```diff
@@ -43,9 +45,12 @@ const defaultProps = {
 function ReferralDetailsPage({route, account}) {
     const theme = useTheme();
     const styles = useThemeStyles();
+    const {windowWidth} = useWindowDimensions();
     const {translate} = useLocalize();
     const popoverAnchor = useRef(null);
     const {isExecuting, singleExecution} = useSingleExecution();
+    const platform = getPlatform();
+    const isNative = platform === CONST.PLATFORM.IOS || platform === CONST.PLATFORM.ANDROID;
```

Comment:
This goes against the coding guidelines as platform-specific files should be used in this instance. `index.native.js` for both iOS and Android, or `index.ios.js` and `index.android.js` respectively if there are different implementations for each native platform.

Reply:
Hmmm... Actually, I've taken this example from another file. Basically, I need only [pass another style](https://github.com/Expensify/App/blob/7b8900108ab9e6ce96e35895ac2541f38b28e081/src/pages/ReferralDetailsPage.js#L72) for one component based on platform. Do you mean create the entire new **ReferralDetailsPage** file to split platforms, because of this 3 lines?

Reply:
Yes, this is correct. You only need to spilt into two files (`index.js` and `index.native.js`) in this case since the IOS and Android implementations would be the same. I don't think there are any exceptions for simple styling differences for this guideline, but maybe @rlinoz can chime in here.

Reply:
Okay, I will split it up.

Reply:
Done - 63cbf53. I've decided to create a separate files for the icon itself, not for the entire screen.

Reply:
Can we make this related to the screen size instead of the platform like we do [here](https://github.com/Expensify/App/blob/5fe1c8aee7ed3638767d7ef53298e01b27c5eeec/src/components/AutoCompleteSuggestions/BaseAutoCompleteSuggestions.tsx#L74) using the [useWindowDimensions](https://github.com/Expensify/App/blob/5fe1c8aee7ed3638767d7ef53298e01b27c5eeec/src/hooks/useWindowDimensions/index.ts#L4)?https://github.com/Expensify/App/blob/5fe1c8aee7ed3638767d7ef53298e01b27c5eeec/src/components/AutoCompleteSuggestions/BaseAutoCompleteSuggestions.tsx#L74

Reply:
Sorry, guys, for this workaround. I've found a way how to do it properly. Since the `viewBox` of the icon is 0 0 `393 232`, the width should be the same - `393px`.

Reply:
Done - 9bffdf7.